src/neo-vm/Types/InteropInterface.cs

```diff
@@ -2,21 +2,21 @@
 
 namespace Neo.VM.Types
 {
-    public class InteropInterface : StackItem
+    public class InteropInterface<T> : StackItem
+        where T : class
     {
-        private IInteropInterface _object;
+        private T _object;
```

Comment:
We could expose this as a public readonly field or property?

Reply:
Why `public`? You can convert it implicitly.

https://github.com/neo-project/neo-vm/blob/c44d737cabff782fb56f49cfc7f4c1ee9ef9bbf0/src/neo-vm/Types/InteropInterface.cs#L33-L36

Reply:
Because i need to get the `object` so we need to call this conversion with reflection.

Reply:
I try to avoid reflection, because failures only appear in unit testing, not during compilation

Reply:
If is needed reflection to get the base object, this change is not needed, because could be extracted with reflection too